server/portal/settings/settings_custom.example.py

```diff
@@ -0,0 +1,237 @@
+"""
+All secret values (eg. configurable per project) - usually stored in UT stache.
+"""
+
+########################
+# DJANGO SETTINGS COMMON
+########################
+
+_DEBUG = True
+
+# Namespace for portal
+_PORTAL_NAMESPACE = 'CEP'
+_PORTAL_DOMAIN = 'Core Portal'
+
+# NOTE: set _WH_BASE_URL to ngrok redirect for local dev testing (i.e. _WH_BASE_URL = 'https://12345.ngrock.io', see https://ngrok.com/)
+_WH_BASE_URL = ''
+
+# Unorganized
+_LOGIN_REDIRECT_URL = '/workbench/dashboard'
+_SYSTEM_MONITOR_DISPLAY_LIST = ['frontera.tacc.utexas.edu', 'stampede2.tacc.utexas.edu',
+                                'maverick2.tacc.utexas.edu', 'longhorn.tacc.utexas.edu']
+
+########################
+# DJANGO SETTINGS LOCAL
+########################
+
+_RT_QUEUE = 'Web & Mobile Apps'
+_RT_TAG = 'core_portal'
+
+########################
+# AGAVE SETTINGS
+########################
+
+_AGAVE_STORAGE_SYSTEM = 'cep.storage.default'
+_AGAVE_DEFAULT_TRASH_NAME = 'Trash'
+
+_AGAVE_JWT_HEADER = 'HTTP_X_JWT_ASSERTION_PORTALS'
+
+########################
+# ELASTICSEARCH SETTINGS
+########################
+
+_COMMUNITY_INDEX_SCHEDULE = {}
+
+########################
+# DJANGO APP: WORKSPACE
+########################
+
+_PORTAL_APPS_METADATA_NAMES = ["portal_apps", "portal_apps_dev"]
```

Comment:
__Question__
Why is `portal_apps(…)` in `settings_custom` and `frontera_apps(…)` is `settings_default`? _Portal seems more like a default project value, and Frontera seems more like a custom project value._

Reply:
This file is simply an example file, not intended to be used but overwritten. `portal` is generic here. 

`settings_default` is intended to act as a local dev file (which I'll rename) - whatever is most useful for devs locally is what I hoped to put here. I think I have more up-to-date apps under the frontera namespace. But it would make sense to be `portal_apps` here too

Reply:
Okay. Just making sure there's reason. I thought it might be that Frontera "had more". 👍